### ISSUE TITLE
Callout text is ALL CAPS

### ISSUE THREAD

**Issue by fherreazcue** [UoMResearchIT/sandpaper-docker#1]
Only seems to be callout- blocks... prerequisites, challenges, and so on seem to be ok...

This is a local render using the docker container:

 <img src="https://github.com/UoMResearchIT/sandpaper-docker/assets/36694679/d044b665-a6ce-4627-8456-196b3a8da055" width=50%>

.

The render produced by carpentries workflows works ok:

<img src="https://github.com/UoMResearchIT/sandpaper-docker/assets/36694679/0d6e73b9-f4f9-4aa3-982c-0675ec023d0c" width=50%>

**Comment by fherreazcue** [comment]
The callout div is being built differently... the main div id is just `callout1`, and the class of the callout content is `section level3 callout-title callout-inner`, and `callout-title` transforms text to all-caps... 

Workflow version:
![Screenshot from 2023-06-28 11-53-47](https://github.com/UoMResearchIT/sandpaper-docker/assets/36694679/7f2e20a6-a5c0-4fb6-88fd-1f6a035a3a35)

Docker version:
![Screenshot from 2023-06-28 11-53-28](https://github.com/UoMResearchIT/sandpaper-docker/assets/36694679/835bef96-522d-4a0f-ba50-4f76987eb340)

Potentially an issue with sandpaper or varnish versions?

**Comment by fherreazcue** [comment]
For some reason the site built using the workflow is using pegboard version 0.5.3:
![Screenshot from 2023-06-28 13-00-31](https://github.com/UoMResearchIT/sandpaper-docker/assets/36694679/ab03e5fb-ecb1-4b56-9897-3b2cc00cbeb8)
Although in theory the [latest release is 0.5.2](https://github.com/carpentries/pegboard/releases)!
![Screenshot from 2023-06-28 13-16-53](https://github.com/UoMResearchIT/sandpaper-docker/assets/36694679/0ea9dde2-7bf1-4165-aade-352acddcb55d)
I don't know how the workflow is getting 0.5.3, so can't try if that solves the problem...

**Comment by fherreazcue** [comment]
The explicit installation of `pegboard` and `tinkr` is not necessary... sandpaper already lists them as prerequisites, so they are installed there... and if removed, it actually installs pegboard `0.5.3` (see [fe343a5](https://github.com/UoMResearchIT/sandpaper-docker/commit/fe343a522623a3a6f94c16315f8d78cabac5dcd9)).

However, this does not solve the problem...

**Comment by fherreazcue** [comment]
Fixed by downgrading pandoc ([dea9c8c](https://github.com/UoMResearchIT/sandpaper-docker/commit/dea9c8c2705e621c91b7f37261b0b802c38ad666)). GH actions uses 2.19.2, and we were using 3.1.2.

**Comment by fherreazcue** [comment]
For future reference:
It was useful to look at the R sessioninfo... the github action lists the package versions installed with sandpaper in the `Setup Lesson Engine` step (near the end, the `Session info` is collapsed).

By running an interactive docker container session, I was able to check packages with different versions by doing:
```
# R
> options(repos = c(
  carpentries = "https://carpentries.r-universe.dev/",
  CRAN = "https://cran.rstudio.com/"
))
> install.packages(c("sessioninfo"))
> sandpaper:::ci_session_info()
```
These packages are even marked, and can be brought to be the same as in the workflow with:
```
> install.packages(c("covr","markdown","mockr","testthat"))
```
But they don't seem to have caused much trouble...

These other packages were also different before this fix:
```
"diffobj","lazyeval","pkgload","praise","rematch2","rex","waldo"
```
The pandoc version is not there, it is used during the serve... it flashes when you try to serve from the interactive container:
```
# R
> sandpaper::serve(host = '0.0.0.0', port = '4321')
── Validating Fenced Divs ──────────────────
── Validating Internal Links and Images ─────────
◉ All files up-to-date; nothing to rebuild!
◉ pandoc found
  version : 3.1.2
...
```
In the GH action, the version can be found in the `Set up Pandoc` step, or in the `Deploy Site` step (in the collapsed `Build Lesson Website` section.)
<img src="https://github.com/UoMResearchIT/sandpaper-docker/assets/36694679/28c41a5c-50a2-4c6a-ad09-e6673e66a65e" width=70%>
